debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -363,6 +363,8 @@ public static SecureConnectionMode parse(String value, String defaultValue) {
     private static final String TABLE_WHITELIST_NAME = "table.whitelist";
     private static final String TABLE_IGNORE_BUILTIN_NAME = "table.ignore.builtin";
 
+    private static final int DEFAULT_BINLOG_BUFFER_SIZE = 1_000;
```

Comment:
I know finding good defaults is hard, but I think we can do a bit more here, e.g. 5.000 or 10.000. The latter would amount to 10MB, assuming an average statement size of 1K. I think that's ok..

Reply:
No problem with it